community-operators/bundles/cluster-logging/bundle.0.0.1.yaml

```diff
@@ -238,7 +238,6 @@ data:
                           description: Number of nodes to deploy
                           format: int32
                           type: integer
-                          minimum: 1
```

Comment:
OLM has resolved the issue. What's our plan to add it back?

Reply:
We never removed these fields ourselves, the cluster-logging vendor took it upon himself to do so. If the issue is resolved, then I can ask him to add it back.